{{cookiecutter.profile_name}}/lsf_submit.py

```diff
@@ -84,21 +72,24 @@ def resources(self) -> dict:
         return self.job_properties.get("resources", dict())
 
     @property
-    def mem_mb(self) -> int:
-        return self.resources.get(
+    def mem_mb(self) -> Memory:
+        mem_value = self.resources.get(
             "mem_mb", self.cluster.get("mem_mb", CookieCutter.get_default_mem_mb())
         )
+        return Memory(mem_value, unit=Unit.MEGA)
 
     @property
-    def memory_units(self) -> str:
-        return self._memory_units.value
+    def memory_units(self) -> Unit:
+        return self._memory_units
 
     @property
     def resources_cmd(self) -> str:
+        mem_in_clusters_units = self.mem_mb.to(self.memory_units)
+        value_to_submit = math.ceil(mem_in_clusters_units.value)
         return (
-            "-M {mem_mb}{units} -n {threads} "
-            "-R 'select[mem>{mem_mb}{units}] rusage[mem={mem_mb}{units}] span[hosts=1]'"
-        ).format(mem_mb=self.mem_mb, threads=self.threads, units=self.memory_units)
+            "-M {mem_mb} -n {threads} "
```

Comment:
This is a bit pedantic, but could we change `mem_mb` to either `value_to_submit` or `value_to_submit_in_cluster_units`  or `ceil_mem_in_clusters_units` or a more descriptive name? `mem_mb` is actually confusing, as it is not necessarily memory in MB here anymore

Reply:
Same for the next two lines

Reply:
Ah, yes, that is a good point. I'll make the change now.